packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -66,4 +67,112 @@ export interface ICheckboxProps extends React.HTMLAttributes<HTMLElement | HTMLI
    * @default 'start'
    */
   boxSide?: 'start' | 'end';
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
+
+  /**
+   * Custom styles for this component
+   */
+  styles?: ICheckboxStyles;
 }
+
+export interface ICheckboxStyles {
+  /**
+   * Style for the root element of the checkbox component in the default enabled/unchecked state.
+   */
+  root?: IStyle;
+
+  /**
+   * Style for the checkbox input HTML element. This is hidden visually.
+   */
+  input?: IStyle;
+
+  /**
+   * Style for the label part (contains the customized checkbox + text) when enabled.
+   */
+  label?: IStyle;
```

Comment:
while I understand this is a <label> element, it is confusing to call it as such in the API because "label" means the text that is attached to the control. We should call this something else.